client/packages/system/src/Patient/Components/ProgramSearchInput/ProgramSearchInput.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
```

Comment:
as per the other search input components. This one is under `Patients` because the options shown are specific to a patient